scripts/update-valid-target.js

```diff
@@ -0,0 +1,64 @@
+const https = require('node:https')
```

Comment:
`fetch` is supported natively since Node.js@18

Ref:
- node.js engine specification: https://github.com/nervosnetwork/neuron/blob/develop/package.json#L18
- compatibility: https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API#browser_compatibility